pkg/api/v1/schema.go

```diff
@@ -19,6 +19,7 @@ type DomainSpec struct {
 	SysInfo *SysInfo `json:"sysInfo,omitempty"`
 	Devices Devices  `json:"devices"`
 	Clock   *Clock   `json:"clock,omitempty"`
+	XML     string   `json:"xml"`
```

Comment:
Could you use the annotation section to the VM metadata section? I guess this is only needed to transport information, we do not yet cover in our spec, or which is not yet translated from oVirt into fields in our spec. Annotations should be a perfect fit for that.